app/main.py

```diff
@@ -1,43 +1,90 @@
-from array import array
+# from array import array
 from typing import Optional
-from setuptools import Require
+# from setuptools import Require
 from fastapi import FastAPI
 from pydantic import BaseModel
 from change_calc import calculate_change
-import requests
+# import requests
 
 app = FastAPI()
 
-transactions = {
-    1: {
-        "item_name": "apple",
-        "price": 125,
-        "currency_inserted": 155
-    }
-}
-
 class Transaction(BaseModel):
     item_name: str
     price: int
     currency_inserted: int
 
+class UpdateTransaction(BaseModel):
+    item_name: Optional[str] = None
+    price: Optional[int] = None
+    currency_insterted: Optional[int] = None
+
+transactions = {
+    1: Transaction(item_name="apple", price=125, currency_inserted=155)
+}
+
 @app.get("/")
 def read_root():
-    return {"Hello": "Please enter your item."}
+    return {
+        "status": "Successful Login",
+        "message": "Hello, please enter your item."
+        }
 
 @app.get("/show-transaction/{transaction_id}")
 def read_item(transaction_id : int):
     if transaction_id not in transactions:
-        return {"Error": "Transaction does not exist"}
+        return {
+            "status": "ERROR: Item Not Found",
```

Comment:
Looks great, I would rename item to transaction but otherwise great!

Reply:
Done! Thank you for your notes!